core/src/main/java/nu/yona/server/analysis/entities/IntervalActivity.java

```diff
@@ -169,6 +164,20 @@ public int getTotalActivityDurationMinutes()
 		return computeTotalActivityDurationMinutes();
 	}
 
+	private List<Integer> spreadBytesAsIntegerList()
+	{
+		if (spread == null)
```

Comment:
This should never be true. The spread should be computed in this case. Actually, it should always have the length of 96.